### DISCUSSION TITLE
GAMES202 - csdiy.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#608] [Announcements]
# GAMES202 - csdiy.wiki

CS自学指南

https://csdiy.wiki/en/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9B%BE%E5%BD%A2%E5%AD%A6/GAMES202/

**Comment by vietnguyenba12** [comment]
Does this course have English subtitle? I can't find any course videos that have English subtitle, so sad I can't understand Chinese

**Reply by indevn** [reply]
If you are a native English speaker, perhaps you can study the courses of UCSD. They teach very well, and you can learn a lot just by looking at the slides. 
UCSD CSE 167 Computer Graphics：https://cseweb.ucsd.edu/~alchern/teaching/cse167_fa21/
UCSD CSE 168 Computer Graphics II: Rendering：https://cseweb.ucsd.edu/~tzli/cse168/sp2023/
UCSD CSE 272: Advanced Image Synthesis：https://cseweb.ucsd.edu/~tzli/cse272/wi2024/
UCSD CSE 274 Selected Topics in Graphics: Sampling and Reconstruction of Visual Appearance: From Denoising to View Synthesis：https://cseweb.ucsd.edu/~viscomp/classes/cse274/fa21/274.html
